library/Zend/File/Transfer/Adapter/Abstract.php

```diff
@@ -1435,7 +1435,7 @@ protected function _getTmpDir()
     protected function _isPathWriteable($path)
     {
         $tempFile = rtrim($path, "/\\");
-        $tempFile .= '/' . 'test.1';
+        $tempFile .= '/' . 'test.' . rand(1, 1000);
```

Comment:
Maybe 10000 or 100000? With 10000 this error will still happen once in a year, at that time we forget what was the reason and someone will spent lot of time finding it :D

Reply:
No problem, I'll update it to 100k :)